### PR TITLE
Improve groovy script generation from filters

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/main/java/org/gridsuite/actions/server/FiltersToGroovyScript.java [hunk 1]
```diff
@@ -80,15 +80,15 @@ public String generateGroovyScriptFromFilters(FiltersContingencyListAttributes f
         ST template = new ST(script);
 
         template.add("collectionName", equipmentsCollection);
+        template.add("nominalVDefined", filtersContingencyListAttributes.getNominalVoltage() != -1);
```
Comment (geofjamg) [inline src/main/java/org/gridsuite/actions/server/FiltersToGroovyScript.java:83]: Was it really necessary to introduce nominalVDefined ? why not just not defined nominalV

File: src/main/java/org/gridsuite/actions/server/FiltersToGroovyScript.java [hunk 2]
```diff
@@ -80,15 +80,15 @@ public String generateGroovyScriptFromFilters(FiltersContingencyListAttributes f
...
         template.add("nominalV", filtersContingencyListAttributes.getNominalVoltage());
         template.add("nominalVOperator", filtersContingencyListAttributes.getNominalVoltageOperator().equals("=") ?
                 "==" :
                 filtersContingencyListAttributes.getNominalVoltageOperator());
         template.add("equipmentId", filtersContingencyListAttributes.getEquipmentID());
         template.add("equipmentName", filtersContingencyListAttributes.getEquipmentName());
-        template.add("countries", filtersContingencyListAttributes.getCountries().isEmpty()
-                ? "[]"
-                : filtersContingencyListAttributes.getCountries().stream().collect(joining("','", "['", "']")));
+        template.add("emptyCountries", filtersContingencyListAttributes.getCountries().isEmpty());
```
Comment (geofjamg) [inline src/main/java/org/gridsuite/actions/server/FiltersToGroovyScript.java:90]: Same

**Review by geofjamg — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/actions/server/FiltersToGroovyScript.java (hunk 1 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/actions/server/FiltersToGroovyScript.java]: Done

File: src/main/java/org/gridsuite/actions/server/FiltersToGroovyScript.java (hunk 2 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/actions/server/FiltersToGroovyScript.java]: Done

**Review by geofjamg — APPROVED** [review]

**Pull request merged**